docs/README.md

```diff
@@ -1,56 +1,56 @@
+ClickHouse uses the "documentation as code" approach, so you can edit Markdown files in this folder from the GitHub web interface. Alternatively, fork the ClickHouse repository, edit, commit, push, and open a pull request.
 
-At the moment documentation is bilingual in English and Russian, so it's better to try keeping languages in sync if you can, but it's not strictly required as there are people watching over this. If you add new article, you should also add it to `toc_{en,ru,zh,fa}.yaml` files with pages index.
+At the moment documentation is bilingual in English and Russian. It's best to try to keep both languages in sync if you can, but this is not strictly required because there are people watching over this. If you add a new article, you should also add it to `toc_{en,ru,zh,fa}.yaml` files with the pages index.
 
-Master branch is then asynchronously published to ClickHouse official website:
+The master branch is then asynchronously published to the ClickHouse official website:
 
 * In English: https://clickhouse.yandex/docs/en/
 * In Russian: https://clickhouse.yandex/docs/ru/
 * In Chinese: https://clickhouse.yandex/docs/zh/
 * In Farsi: https://clickhouse.yandex/docs/fa/
 
-Infrastructure to build Markdown to documentation website resides in [tools](tools) folder, it has it's own [README.md](tools/README.md) with more details.
+The infrastructure to build Markdown for publishing on the documentation website resides in the [tools](tools) folder. It has its own [README.md](tools/README.md) file with more details.
 
-# How to write content for ClickHouse documentation?
+# How to Write Content for ClickHouse Documentation
 
-## Target audience
+## Target Audience
 
-When you write pretty much any text, first thing you should think about: who exactly will read it and in which terms it is better to "talk" with them.
+When you write pretty much any text, the first thing you should think about is who will read it and which terms you should use for communicating with them.
 
-ClickHouse can be directly used by all sorts of either analysts and engineers, so you should only basic technical background of reader when writing content for generic parts of documentation, like query language, tutorials or overviews. Though it is ok for articles describing ClickHouse internals, guides for operating ClickHouse clusters, contributing to C++ code and other similar topics.
+ClickHouse can be directly used by all sorts of analysts and engineers. For generic parts of documentation (like the query language, tutorials or overviews), assume that the reader only has a basic technical background. For more technical sections (like articles that describe ClickHouse internals, guides for operating ClickHouse clusters, or rules for contributing to C++ code), you can use specific technical language and concepts.
```

Comment:
I'd replace "specific" with some synonym(s) because next section is called "Specific ..."